src/Microsoft.DotNet.Interactive.Tests/Server/SerializationTests.cs

```diff
@@ -134,11 +161,26 @@ public void All_command_types_are_tested_for_round_trip_serialization()
         [Fact]
         public void All_event_types_are_tested_for_round_trip_serialization()
         {
-            var eventTypes = typeof(KernelEvent)
-                             .Assembly
-                             .ExportedTypes
-                             .Concrete()
-                             .DerivedFrom(typeof(KernelEvent));
+
+            var interactiveEvents = typeof(Kernel)
```

Comment:
Don't we discover the others using reflection? We should do the same for these.

Reply:
what do you mean, they are discovered scanning the assemblies using reflection